src/components/profile/ProfileService.js

```diff
@@ -331,6 +468,20 @@ goog.require('ga_urlutils_service');
                   .tickSize(-width, 0, 0)
                   .tickFormat('')
               );
+          this.group.select('text.ga-profile-elevation-difference')
+            .transition().duration(transitionTime)
+              .text('\u03A3 \u0394 ' + $translate.instant(options.yLabel) +
+                  ': ' + indicatorFilter(Math.round(this.diff * 100) / 100));
+          this.group.select('text.ga-profile-elevation-up')
+            .transition().duration(transitionTime)
+              .text('\u03A3 + ' + $translate.instant(options.yLabel) +
+                  ': ' + indicatorFilter(Math.round(this.twoDiff[0] * 100) /
+                  100));
+          this.group.select('text.ga-profile-elevation-down')
+            .transition().duration(transitionTime)
+              .text('\u03A3 - ' + $translate.instant(options.yLabel) +
+                  ': ' + indicatorFilter(Math.round(this.twoDiff[1] * 100) /
+                  100));
```

Comment:
If we don't need the transition stuf I think you can create and `updateElevationLabels(diff, twoDiff)` to avoid duplicate code with [line 386](https://github.com/geoadmin/mf-geoadmin3/pull/2754/files#diff-bfd39f2f09138781ec89cd2819dbd0d7R386)